app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power,
+                 average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        resultlist = []
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                resultlist.append(self.calculate_washing_price(car))
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                self.wash_single_car(car)
```

Comment:
Why do you have 2 similar loops? You can calculate price and wash in one loop

Reply:
I do it to avoid changing car.clean_mark that I use for self.calculate_washing_price(car). Thant's why at first I calculate with old value of car.clean_mark and THEN I use self.wash_single_car(car) where car.clean_mark changes

Reply:
I understood.